code/datums/components/pact_weapon.dm

```diff
@@ -62,10 +62,10 @@
 /datum/component/pact_weapon/proc/equipped(obj/item/source, mob/user, slot)
 	var/mob/living/target = user
 	if(target != weapon_owner) //you dont own the weapon
-		to_chat(weapon_owner, span_warning("[target] has equipped [weapon]!")) //message the rightful owner 
-		to_chat(target, span_warning("[weapon] burns you as you equip it!")) //message the wielder
-		target.apply_status_effect(/datum/status_effect/buff/pact_weapon_debuff) //apply debuff to wielder
-
+		to_chat(weapon_owner, span_warning("[target] tried to equip [weapon]!")) //message the rightful owner 
+		to_chat(target, span_danger("[weapon] slips from your grasp!")) //message the wielder
+		target.Stun(10) //this is not yours, drop it
```

Comment:
Change target.Stun(10) to target.dropItemToGround(src), as this achieves the desired effect without requiring a stun.

Reply:
Yeah that makes more sense.